io/zenoh-transport/tests/unicast_transport.rs

```diff
@@ -375,7 +404,14 @@ async fn open_transport(
     // Open transport -> This should be accepted
     for e in client_endpoints.iter() {
         println!("Opening transport with {}", e);
-        let _ = ztimeout!(client_manager.open_transport(e.clone())).unwrap();
+        //let _ = ztimeout!(client_manager.open_transport(e.clone())).unwrap();
```

Comment:
Don't remove the `ztimeout!`.

Reply:
That was just the debug code slipped here, sorry